next.config.js

```diff
@@ -0,0 +1,9 @@
+
+const { PHASE_DEVELOPMENT_SERVER } = require('next/constants')
+
+module.exports = (phase, { defaultConfig }) => {
+  if (phase === PHASE_DEVELOPMENT_SERVER) {
+    require('dotenv').config()
+    return defaultConfig
+  }
+}
```

Comment:
Sorry - I am not super familiar with NextJS yet, but what is the purpose of this file? Just curious, because it seems like previous code worked as is without one.

Reply:
The `require('dotenv').config()` can go above the module.exports or be dropped altogether. It depends on whether `.env` is expected to work in production. I prefer it not.

https://nextjs.org/docs/api-reference/next.config.js/introduction

Reply:
Ahh I see! 😮